### PR TITLE
LRDOCS-5610 Overriding JSPs using OSGi fragments and custom JSP bags

### PR DESCRIPTION
**Pull request by jhinkey**
Hi Ray,

I've ported these tutorials from 7.0 to 7.1. Would you review or delegate their review?

https://issues.liferay.com/browse/LRDOCS-5610

Thanks,
Jim

### CODE REVIEW

**Comment by rotty3000** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: develop/tutorials/articles/120-customizing/01-customizing-jsps/05-core-jsp-overrides-using-customjspbag.markdown [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+Register the custom JSP bag implementation from your module in the OSGi runtime
+with three properties:
+
+-  **`context.id`:** Specify your custom JSP bag class name. For example,
+   `MyCustomJspBag`.
+
+-  **`contex.name`:** This should be a more human readable name, like `My Custom
+   JSP Bag`.
+
+-  **`service.ranking:integer`:** This will determine the priority of your
```
Comment (rotty3000) [inline develop/tutorials/articles/120-customizing/01-customizing-jsps/05-core-jsp-overrides-using-customjspbag.markdown]: `service.ranking:Integer` (capital `I`)

**Comment by jhinkey** [comment]
Great! I capitalized "Integer".

**Comment by jhinkey** [comment]
Sent https://github.com/sez11a/liferay-docs/pull/3525

**Pull request merged**